### PR TITLE
Add block patterns notice

### PR DESCRIPTION
**Pull request by TeBenachi**
#295

### CODE REVIEW

File: checks/block-patterns.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+				__( 'RECOMMENDED', 'theme-check' ),
+				__( 'No reference to <strong>register_block_pattern</strong> was found in the theme. Theme authors are encouraged to implement custom block patterns as a transition to block-based themes.', 'theme-check' )
+			);
+		}
+
+		if ( strpos( $php, 'register_block_style' ) === false ) {
+			$this->error[] = sprintf(
+				'<span class="tc-lead tc-recommended">%s</span>: %s',
+				__( 'RECOMMENDED', 'theme-check' ),
+				__( 'No reference to <strong>register_block_style</strong> was found in the theme. Theme authors are encouraged to implement new block styles as a transition to block-based themes.', 'theme-check' )
```
Comment (carolinan) [inline checks/block-patterns.php:24]:
```suggestion
				__( 'No reference to <strong>register_block_style</strong> was found in the theme. Theme authors are encouraged to implement new block styles as a transition to block themes.', 'theme-check' )
```

**Comment by carolinan** [comment]
Thank you.
I only made two small changes, we are recommended to use the term block themes instead of block-based themes.

**Comment by carolinan** [comment]
We can also add links to the documentation for the functions later.

**Comment by TeBenachi** [comment]
Thank you!

**Pull request merged**